charm-slurmctld/src/interface_slurmctld_peer.py

```diff
@@ -131,8 +140,8 @@ def _on_relation_changed(self, event):
             ctxt["active_controller_ingress_address"] = unit_relation_data[
                 "ingress-address"
             ]
-            ctxt["active_controller_hostname"] = self._charm.get_hostname()
-            ctxt["active_controller_port"] = str(self._charm.get_port())
+            ctxt["active_controller_hostname"] = self._charm.hostname
+            ctxt["active_controller_port"] = str(self._charm.port)
```

Comment:
we use `str()` here but not in other places.... I think its ok to remove the extra string formatting as the value should already be a string